### PR TITLE
Thankyou fix

### PR DESCRIPTION
**Pull request by OlaShabalina**
## Overview of the Pull Request:
Update content of the thank you page.
Required: 
image
Now (before changes): 
image

## link to Trello card or Github issue
(https://trello.com/c/DS2GOKNQ)

## How Has This Been Tested?
Visual

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [x] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [x] Is your pull request up-to-date with `main` branch?
- [x] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [x] Have you written instructions on how to test that your changes work as expected?
- [x] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

**Comment by OlaShabalina** [comment]
New update to design from **Zara** and **Sean Si**: text is now aligned with the heading 😄
image

File: pages/form/Thankyou.jsx [hunk 1]
```diff
@@ -8,43 +8,55 @@ export default function Thankyou() {
...
+            Thanks for participating, your time is greatly appreciated!
+          </Text>
+          <Text
+            color="#fff"
+            mt={5}
+            fontSize="18px"
+            textAlign={["center", "start"]}
+            w="100%"
+          >
+            We’re collecting responses from 26th April 2022 to better understand
```
Comment (hqtan) [inline pages/form/Thankyou.jsx:35]: We will probably need to change the date from 26th April to something else? (probably need to confirm with Sean)

```suggestion
            We’re collecting responses from 26th April 2022 to better understand 
```

File: pages/form/Thankyou.jsx [hunk 2]
```diff
@@ -8,43 +8,55 @@ export default function Thankyou() {
...
+          <Text
+            color="#fff"
+            mt={5}
+            fontSize="18px"
+            textAlign={["center", "start"]}
+            w="100%"
+          >
+            As part of Climate Change and Environment Strategy (2021 to 2026), 
+            we’ve set a challenge that 20% of staff trips to work are by active, 
+            shared and public transport by 2026. So I’ll be interesting to see where
```
Comment (hqtan) [inline pages/form/Thankyou.jsx:47]: minor typo to fix
```suggestion
            shared and public transport by 2026. So It’ll be interesting to see where 
```

File: pages/form/Thankyou.jsx (hunk 1 above)
Comment (OlaShabalina) [reply pages/form/Thankyou.jsx:35]: Agree, we can add the current one for now and change the date later, what do you think? At the moment the text and style are very different. 😄

File: pages/form/Thankyou.jsx (hunk 2 above)
Comment (OlaShabalina) [reply pages/form/Thankyou.jsx:47]: Wow, this is a good one, need to change in the design as well! 👍

**Review by hqtan — APPROVED** [review]
nice work

**Pull request merged**